fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -161,6 +163,9 @@ struct MetaclusterManagementWorkload : TestWorkload {
 		try {
 			state metacluster::DataClusterEntry entry;
 			entry.capacity.numTenantGroups = deterministicRandom()->randomInt(0, 4);
+			if (deterministicRandom()->randomInt(0, 4) == 0) {
```

Comment:
Nit: this should also be specified in terms of probabilities

Reply:
Done